config.sample.yml

```diff
@@ -117,7 +117,7 @@ encryption:
   # (Optional) Configuration for encryption support in the bridge.
   # If omitted, encryption support will be disabled.
   #
-  storagePath: ./data/encryption
+  null
```

Comment:
How about commenting out the whole section?  I can't remember if the code chokes on having an `encryption` object with no properties, vs an undefined `encryption` object altogether

Reply:
The null was a bit of a mistake. It was meant to leave this out of the config. We don't yet have support for generating commented out sections (though it's something I want to figure out eventually)